### PR TITLE
Wiki Reorg: Add Constants and Variables Page (Learn Section)

### PR DESCRIPTION
**Pull request by filippoweb3**
Remove all rpc calls fetching variables on all .md files and centralizing them in one single page for both Polkadot, Kusama and system chains where applicable. No conditional rendering, but tabs.

### CODE REVIEW

**Comment by filippoweb3** [comment]
@CrackTheCode016 asking for a preliminary review. Let me know wdyt and I will continue with this.

File: docs/general/constants-variables.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#### Minimum Active Bond
+
+The minimum amount of tokens to nominate is <RPC network="polkadot" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>.
+
+</TabItem>
+<TabItem value="kusama">
+
+#### Treasury Spending Period
+
+The spending period on Polkadot is currently <RPC network="kusama" path="consts.treasury.spendPeriod" defaultValue={86400} filter="blocksToDays"/> days.
```
Comment (CrackTheCode016) [inline docs/general/constants-variables.md]:
```suggestion
The spending period on Kusama is currently <RPC network="kusama" path="consts.treasury.spendPeriod" defaultValue={86400} filter="blocksToDays"/> days.
```

File: docs/general/constants-variables.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+#### Minimum Bond to Participate in Staking
+
+The minimum bond to nominate on Polkadot is <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> while the minimum amount to join a pool is <RPC network="kusama" path="query.nominationPools.minJoinBond" defaultValue={1667000000} filter="humanReadable"/>.
```
Comment (CrackTheCode016) [inline docs/general/constants-variables.md]:
```suggestion
The minimum bond to nominate on Kusama is <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> while the minimum amount to join a pool is <RPC network="kusama" path="query.nominationPools.minJoinBond" defaultValue={1667000000} filter="humanReadable"/>.
```

**Review by CrackTheCode016 — COMMENTED** [review]
I like how it is coming together so far, much cleaner!

I see it as a good bridge towards upgrading to docusaurus v3, and post upgrade, perhaps that's when we can explore other ways to integrate RPC info in the wiki. For now, this works well though, and we can always just link to this page wherever it is applicable.

File: docs/general/constants-variables.md [hunk 3]
```diff
@@ -0,0 +1,419 @@
+---
+id: constants-variables
```
Comment (profradha) [inline docs/general/constants-variables.md]: **variables** is a generic mathematical term not often used in the blockchain context. It also has a different meaning in programming context when compared to what is listed on this page. Every item on this page is a queryable chain constant or a storage value, so I suggest to rename to `chain-constants-storage`
 
 We can also use `chain-state-values` or something along those lines.
 

```suggestion
id: chain-constants-storage
```

File: docs/general/constants-variables.md [hunk 4]
```diff
@@ -0,0 +1,419 @@
...
+title: Constants and Variables
```
Comment (profradha) [inline docs/general/constants-variables.md]:
```suggestion
title: Chain Constants and Storage Values
```

File: docs/general/constants-variables.md [hunk 5]
```diff
@@ -0,0 +1,419 @@
...
+sidebar_label: Constants and Variables
```
Comment (profradha) [inline docs/general/constants-variables.md]: We can find a better sidebar label. 

But **variables** is a generic mathematical term. Every item on this page is a queryable chain constant or a storage value. "Chain State Values" could be a shorter version of this title.

```suggestion
sidebar_label: Chain Constants and Storage Values
```

File: docs/general/constants-variables.md [hunk 6]
```diff
@@ -0,0 +1,419 @@
...
+description: Constants and Variables.
```
Comment (profradha) [inline docs/general/constants-variables.md]:
```suggestion
description: Chain constants and storage values that can be queried from the live network nodes of Polkadot, Kusama and their system chains.
```

File: docs/general/constants-variables.md [hunk 7]
```diff
@@ -0,0 +1,419 @@
...
+keywords: [polkadot, kusama, constants, variables]
```
Comment (profradha) [inline docs/general/constants-variables.md]:
```suggestion
keywords: [polkadot, kusama, constants, storage, values]
```

File: docs/general/constants-variables.md [hunk 8]
```diff
@@ -0,0 +1,419 @@
...
+
+import RPC from "./../../components/RPC-Connection"; import Tabs from "@theme/Tabs"; import TabItem
+from "@theme/TabItem";
+
+<!-- prettier-ignore-start -->
+<Tabs groupId="chains" values={[ {label: 'Polkadot', value: 'polkadot'}, { label: 'Kusama', value: 'kusama'}, {label: 'Polkadot Asset Hub', value: 'ahp'}, {label: 'Kusama Asset Hub', value: 'ahk'}, {label: 'Polkadot People', value: 'pp'}, {label: 'Kusama People', value: 'kp'} ]}>
+
+<TabItem value="polkadot">
+
+:::info What to do with my DOT
```
Comment (profradha) [inline docs/general/constants-variables.md]:
```suggestion
:::info What to do with DOT
```

File: docs/general/constants-variables.md [hunk 9]
```diff
@@ -0,0 +1,419 @@
...
+
+Polkadot's inactive issuance is <RPC network="polkadot" path="query.balances.inactiveIssuance" defaultValue="1784854324418488473" filter= "humanReadable"/> in the era <RPC network="polkadot" path="query.staking.currentEra" defaultValue="1200"/>.
+
+#### Index Deposit
+
+The deposit to reserve an index on Polkadot is <RPC network="polkadot" path="consts.indices.deposit" defaultValue={100000000000} filter="humanReadable"/>.
+
+#### Maximum Number of Nominators
+
+The maximum number of nominators on Polkadot is uncapped and currently sits at <RPC network="polkadot" path="query.staking.counterForNominators" defaultValue={50000}/>.
```
Comment (profradha) [inline docs/general/constants-variables.md]:
```suggestion
The maximum number of nominators on Polkadot is uncapped and the current value is <RPC network="polkadot" path="query.staking.counterForNominators" defaultValue={50000}/>.
```

File: docs/general/constants-variables.md [hunk 10]
```diff
@@ -0,0 +1,419 @@
...
+
+The maximum number of proxies per Polkadot account is <RPC network="polkadot" path="consts.proxy.maxProxies" defaultValue={32}/>. You can have the same proxy for multiple accounts.
+
+#### Maximum Votes per Nominator
+
+A nominator on Polkadot can select up to <RPC network="polkadot" path="consts.electionProviderMultiPhase.minerMaxVotesPerVoter" defaultValue={16} /> validators.
+
+#### Minimum Active Bond
+
+The minimum amount of tokens to nominate on Polkadot is <RPC network="polkadot" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>.
```
Comment (profradha) [inline docs/general/constants-variables.md]:
```suggestion
The minimum amount of DOT required to nominate on Polkadot is <RPC network="polkadot" path="query.staking.minimumActiveStake" defaultValue={2937000000000} filter="humanReadable"/>.
```

File: docs/general/constants-variables.md [hunk 11]
```diff
@@ -0,0 +1,415 @@
...
+The spending period on Polkadot is currently <RPC network="polkadot" path="consts.treasury.spendPeriod" defaultValue={345600} filter="blocksToDays"/> days.
+
+#### Unbonding Duration
+
+The unbonding duration on Polkadot is set to <RPC network="polkadot" path="consts.staking.bondingDuration" defaultValue={28} filter="erasToDays"/> days.
+
+</TabItem>
+<TabItem value="kusama">
+
+:::info What to do with my KSM
```
Comment (profradha) [inline docs/general/constants-variables.md]:
```suggestion
:::info What to do with KSM
```

File: docs/general/constants-variables.md [hunk 12]
```diff
@@ -0,0 +1,415 @@
...
+#### Sub-identity Deposit
+
+The creation of a Polkadot sub-identity required a deposit of <RPC network="polkadotpeople" path="consts.identity.subAccountDeposit" defaultValue={2005300000} filter="humanReadable"/>.
+
+</TabItem>
+<TabItem value="kp">
+
+#### Identity Deposit
+
+The creation of a Polkadot identity required a deposit of <RPC network="kusamapeople" path="consts.identity.basicDeposit" defaultValue={6672333321} filter="humanReadable"/> and <RPC network="kusamapeople" path="consts.identity.byteDeposit" defaultValue={333333} filter="humanReadable"/> (per byte) per each field beyond the legal name.
```
Comment (profradha) [inline docs/general/constants-variables.md]:
```suggestion
The creation of a Kusama identity required a deposit of <RPC network="kusamapeople" path="consts.identity.basicDeposit" defaultValue={6672333321} filter="humanReadable"/> and <RPC network="kusamapeople" path="consts.identity.byteDeposit" defaultValue={333333} filter="humanReadable"/> (per byte) per each field beyond the legal name.
```

File: docs/general/constants-variables.md [hunk 13]
```diff
@@ -0,0 +1,415 @@
...
+
+#### Sub-identity Deposit
+
+The creation of a Polkadot sub-identity required a deposit of <RPC network="kusamapeople" path="consts.identity.subAccountDeposit" defaultValue={6684333309} filter="humanReadable"/>.
```
Comment (profradha) [inline docs/general/constants-variables.md]:
```suggestion
The creation of a Kusama sub-identity required a deposit of <RPC network="kusamapeople" path="consts.identity.subAccountDeposit" defaultValue={6684333309} filter="humanReadable"/>.
```

**Review by DrW3RK — COMMENTED** [review]
Shared feedback on the page title and description. 

Some of the observations

- Would it be possible to link to the Kusama values from other Wiki pages? For instance, there is a spending period link from the glossary which by default links to Polkadot values. What if there is a situation where we would like to link to a Kusama value?

- The milliDOT looks like the odd man out in the What to do with DOT section. It is even more prominent with milliKSM where other values are shown in KSM with decimals and only identity is shown in milli values. Standardizing to just DOT or KSM provides uniformity and ease of comparison.

File: docs/general/constants-variables.md (hunk 10 above)
Comment (filippoweb3) [reply docs/general/constants-variables.md]: DOT already specified at the end of the sentence

File: docs/general/constants-variables.md (hunk 9 above)
Comment (filippoweb3) [reply docs/general/constants-variables.md]: implemented

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**